### ISSUE TITLE
Steam Experimental 23/09 update appears incompatible

### ISSUE THREAD

**Issue by Devilin-Pixy** [goatfungus/NMSSaveEditor#1085]
A quick heads up for those on Steam Experimental !!!
Saves from today's [Steam Experimental update 23/09/2024](https://steamcommunity.com/app/275850/discussions/0/4845399912177586938/) fail to load with NMSSE 1.17.5
[Steam Experimental **15794149**](https://steamdb.info/app/275850/history/?changeid=25349163) (SteamDB :link:) BuildID: **15794149**

[NMSSaveEditor.log](https://github.com/user-attachments/files/17101402/NMSSaveEditor.log)
Slot 4 and Slot 5 both fail, after having saved in experimental. It appears to fail on the Steam metadata files (header format)

**Comment by boldfish** [comment]
Confirmed issue.  Once a slot has been saved in game, it no longer appears selectable in the editor.

Since it's 'experimental' that's affected, i'm going to assume we have to wait now until the new 'save file' improvements make their way to the main branch and the editor can be updated.

Playing 'vanilla' is good for the soul, but not for the impatient...

**Comment by emilio-simoes** [comment]
I have the same issue, is definitely something related with the metadata header format:

```
[INFO] Reading metadata for save2.hg
[WARNING] cannot read file metadata: mf_save2.hg
[WARNING] java.io.IOException: Invalid or unsupported format in metadata header: 2004
	at nomanssave.fI.a(Unknown Source)
	at nomanssave.fI.b(Unknown Source)
	at nomanssave.fP.<init>(Unknown Source)
	at nomanssave.fL.<init>(Unknown Source)
	at nomanssave.fI.<init>(Unknown Source)
	at nomanssave.fq.a(Unknown Source)
	at nomanssave.Application.<init>(Unknown Source)
	at nomanssave.Application.<init>(Unknown Source)
	at nomanssave.w.run(Unknown Source)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
```

**Comment by Devilin-Pixy** [comment]
With the new update for the expedition, this means the experimental changes from a month ago, have now gone public. Any save with current versions will not work until a fix for this editor is pushed.

**Comment by PappyVanWink** [comment]
> With the new update for the expedition, this means the experimental changes from a month ago, have now gone public. Any save with current versions will not work until a fix for this editor is pushed.

I know it's a big ask but I hope we get an update soon as this is a really short expedition.

**Comment by emilio-simoes** [comment]
yeah! don't wait for any action from these guys until the expedition is done, if any (they don't want to make it unfair to other players!?!?!?). Otherwise they would make the source code available for PR's from other developers who could help and provide quicker fixes.

**Comment by Devilin-Pixy** [comment]
Issue appears to have been fixed with update 1.17.6.
A shame the experimental that broke it, took so long to go public only now with the expedition release.

Thanks @goatfungus , much appreciated, I'll close this one.
